pkg/controller/dnszone/dnszone_controller.go

```diff
@@ -50,6 +51,9 @@ var (
 	},
 		[]string{"force"},
 	)
+
+	// reg is a regex used to fetch condition message from error when credentials are missing permission to create DNS zone
+	reg = regexp.MustCompile(`^AccessDeniedException:\s+([^\t]+)\n\t`)
```

Comment:
For a package-wide constant/var can we give it a more descriptive variable name? Maybe `accessDeniedMessageRegexp`?